client/plugins/os_updates.py

```diff
@@ -179,9 +181,21 @@ def status(current, updates):
             return status(True, False)
         log.error('Call to pacman -Qu failed. Output: {}',
                   e.output.decode('utf8'))
-        return status(True, 'unknown')
+        return status(True, True, 'unknown')
```

Comment:
why does `pacman -Qu` failing now imply that `updates=True`?

Reply:
Good catch. Fixed.